static/sass/_pattern_navigation.scss

```diff
@@ -57,7 +57,12 @@ $meganav-height: 3rem;
     }
 
     .p-link--inverted {
+      color: $color-x-light;
```

Comment:
I added this as AFAIK the design for things like the meganav hasn't changed, the links should not be blue as they are when using the is-dark class. If they are indeed supposed to be blue now, please let me know and I'll remove